ddp-workspace/projects/ddp-rgp/src/app/constants/study-messages.ts

```diff
@@ -13,14 +13,7 @@ export const studyMessagesConfiguration: StudyMessageConfiguration[] = [
    */
   {
     workflowKey: WorkflowKey.AcceptanceStatus,
-    condition: workflow => workflow.status === AcceptanceStatus.PreReview,
-    baseKey: 'Application',
-    stageKey: 'PreReview',
-    group: 1,
-  },
-  {
-    workflowKey: WorkflowKey.AcceptanceStatus,
-    condition: workflow => workflow.status === AcceptanceStatus.InReview,
+    condition: workflow => workflow.status === AcceptanceStatus.PreReview || workflow.status === AcceptanceStatus.InReview,
```

Comment:
There's some issues with changing this to both InReview and PreReview, so let's revert this until we get more clarification.